Makefile

```diff
@@ -79,6 +80,18 @@ mmusdv-base.owl: mmusdv.owl
 		--trim false \
 		--output $@
 
+foodon.owl:
+	curl -L -O 'http://purl.obolibrary.org/obo/mmusdv.owl'
```

Comment:
A typo, I assume?

Reply:
I fixed it right after merging https://github.com/INCATools/ubergraph/commit/17cad5a4445e492ef8ac2b431649f1d744a28ec8 😆